app/src/app-state-reducer.js

```diff
@@ -12,23 +12,47 @@ export default function reducer(state) {
     }
   }
 
-  const { proposals, stakeToken } = state
+  const {
+    proposals,
+    stakeToken,
+    globalParams,
+    requestToken,
+    convictionStakes,
+  } = state
+
+  const pctBaseBN = new BigNumber(globalParams.pctBase.toString())
+
   return {
     ...state,
-
+    globalParams: {
+      alpha: new BigNumber(globalParams.decay.toString()).div(pctBaseBN),
+      maxRatio: new BigNumber(globalParams.maxRatio.toString()).div(pctBaseBN),
+      weight: new BigNumber(globalParams.weight.toString()).div(pctBaseBN),
```

Comment:
i think that there is nothing wrong about sending the right data type to the library instead of relying on what they are going to convert it, if for some reason they decide to do a breaking change and not convert it anymore we are still compatible with them with no change

Reply:
Fair enough